release/models/network-instance/openconfig-evpn.yang

```diff
@@ -40,9 +40,15 @@ module openconfig-evpn {
     domains, this is not currently supported and requires an extension
     of the model.";
 
-  oc-ext:openconfig-version "0.4.0";
+  oc-ext:openconfig-version "1.0.0";
```

Comment:
I suggest we only increment to 0.5.0, as the evpn model is probably not ready to be v1.0.0 yet?